compiler/src/iree/compiler/Codegen/Common/EncodingUtils.cpp

```diff
@@ -64,7 +65,11 @@ MaterializeEncodingConversionTarget::MaterializeEncodingConversionTarget(
   markUnknownOpDynamicallyLegal([](Operation *op) {
     auto typeHasEncoding = [](Type t) -> bool {
       auto tensorType = dyn_cast<RankedTensorType>(t);
-      return tensorType && tensorType.getEncoding();
+      if (!(tensorType && dyn_cast_or_null<IREE::Encoding::EncodingAttr>(
+                              tensorType.getEncoding())))
+        return false;
+      // Allow iree_encoding::packed_storage to pass through.
+      return !IREE::Encoding::hasPackedStorageAttr(tensorType);
```

Comment:
sorry - my comment should be clearer. I think it can be

```cpp
if (!tensorType) {
  return false;
}
return getEncodingAttr(tensorType) ? true : false;
```

And we probably want to rename the `typeHasEncoding` to `typeHasEncodingForDataTiling`.

Reply:
Oh I see ... For a while I confused `IREEEncoding_Attr` with `EncodingAttr`... One of the names needs to be change to avoid those confusion. :-(